vizro-core/docs/pages/user_guides/pages.md

```diff
@@ -9,7 +9,7 @@ The [`Dashboard`][vizro.models.Dashboard] model accepts the `pages` argument, wh
 
 A [`Page`][vizro.models.Page] is split up into three main containers:
 
-1. The **control container**  where you can add your [Controls](control.md) to interact with the dashboard
+1. The **control container**  where you can add your `controls` (see e.g. [Filters](filters.md) or [Parameters](parameters.md)) to interact with the dashboard
```

Comment:
@nadijagraca I think this needs to be updated as we now have dedicated navigation right?

Reply:
Could you add this to your PR on the images?